membrain.cabal

```diff
@@ -50,6 +50,8 @@ common common-options
                        -fhide-source-paths
                        -Wmissing-export-lists
                        -Wpartial-fields
+if impl(ghc >= 8.8.1)
+  ghc-options:         -Wmissing-deriving-strategies
```

Comment:
Oh, I know why CI failing. I see that indentation went incorrect here, so this conditional is applied to the rest of the stanza instead. You should move it 2 spaces right:

```
  if impl(ghc >= 8.8.1)
    ghc-options:       -Wmissing-deriving-strategies
```

Reply:
Yes, I saw that just now. Sorry about that. Fixed it.